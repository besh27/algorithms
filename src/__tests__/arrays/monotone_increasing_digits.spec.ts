import monotoneIncreasingDigits from '../../algorithms/arrays/monotone_increasing_digits';

describe('Monotone SubArray Test', () => {
  it('should render the correct output via snapshot', () => {
    const numbers = [1, 10, 12, 3, 4, 5, 2, 7, 3, 14, 2];
    expect(monotoneIncreasingDigits(numbers)).toMatchSnapshot();
  });

  it('should render the correct output via snapshot', () => {
    const numbers = [10, 9, 8, 7, 6, 5, 4, 3, 2, 1];
    expect(monotoneIncreasingDigits(numbers)).toMatchSnapshot();
  });

  it('should render the correct output via snapshot', () => {
    const numbers = [1, 5, 10, 7, 15, 12, 17, 16, 20];
    expect(monotoneIncreasingDigits(numbers)).toMatchSnapshot();
  });
});