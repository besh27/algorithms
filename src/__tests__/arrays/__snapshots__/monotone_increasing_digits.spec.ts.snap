// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Monotone SubArray Test should render the correct output via snapshot 1`] = `1e+23`;

exports[`Monotone SubArray Test should render the correct output via snapshot 2`] = `10000000000000000000`;

exports[`Monotone SubArray Test should render the correct output via snapshot 3`] = `1e+22`;
